### PR TITLE
[DE] Update `People/osu! team`

### PR DESCRIPTION
**Pull request by The-Last-Cookie**
I hope there is nothing that I have overlooked. [The diff](https://osu.wiki/status/diff-de-wiki-People-osu!_team) is quite messy.

## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

File: wiki/People/osu!_team/de.md [hunk 1]
```diff
@@ -18,60 +16,74 @@ Die unten aufgeführten Personen bilden den Kern des **osu!-Teams** und sind hau
...
-| ::{ flag=MX }:: [Repflez](https://osu.ppy.sh/users/201392) | Allgemeiner Hausmeister, Wiki-Wartung |
+| ::{ flag=DE }:: [Okoratu](https://osu.ppy.sh/users/1623405) | [Moderator](/wiki/People/Global_Moderation_Team), Berater für die Modding- und Mapping-Szene |
+| ::{ flag=US }:: [pishifat](https://osu.ppy.sh/users/3178418) | Moderator des [osu! YouTube-Kanals](https://www.youtube.com/@osugame), [Mappers' Guild](/wiki/Community/Mappers_Guild) Organisator, [Featured Artist](/wiki/People/Featured_Artists) Kontaktaufnahme |
+| ::{ flag=US }:: [radar](https://osu.ppy.sh/users/7131099) | Leiter des [NATs](/wiki/People/Nomination_Assessment_Team) |
+| ::{ flag=MX }:: [Repflez](https://osu.ppy.sh/users/201392) | [Moderator](/wiki/People/Global_Moderation_Team), [Wiki-Betreuer](/wiki/People/osu!_wiki_maintainers) |
 | ::{ flag=FR }:: [Shiro](https://osu.ppy.sh/users/113005) | Allgemeiner Hausmeister |
 | ::{ flag=AU }:: [smoogipoo](https://osu.ppy.sh/users/1040328) | osu!-Entwickler, osu!mania-Liebhaber, Fehlerbeseitiger |
+::{ flag=PL }:: [spaceman_atlas](https://osu.ppy.sh/users/3035836) | osu!-Entwickler |
```
Comment (The-Last-Cookie) [inline wiki/People/osu!_team/de.md:41]:
```suggestion
| ::{ flag=PL }:: [spaceman_atlas](https://osu.ppy.sh/users/3035836) | osu!-Entwickler |
```

File: wiki/People/osu!_team/de.md [hunk 2]
```diff
@@ -18,60 +16,74 @@ Die unten aufgeführten Personen bilden den Kern des **osu!-Teams** und sind hau
 | :-- | :-- |
 | ::{ flag=AU }:: **[peppy](https://osu.ppy.sh/users/2)** | **Projektleiter** |
 | ::{ flag=GB }:: [-Mo-](https://osu.ppy.sh/users/2202163) | Berater für die Modding- und Mapping-Szene |
-| ::{ flag=CA }:: [Azer](https://osu.ppy.sh/users/2155578) | Organisator und Verwalter des World Cups |
-| ::{ flag=US }:: [ChillierPear](https://osu.ppy.sh/users/9501251) | Organisator und Verwalter des World Cups |
-| ::{ flag=AU }:: [Ephemeral](https://osu.ppy.sh/users/102335) | Projekt- und Community-Betreuer, Featured Artist Kontaktaufnahme |
+| ::{ flag=CA }:: [Azer](https://osu.ppy.sh/users/2155578) | Organisator und Verwalter des [World Cups](/wiki/Tournaments#offizielle-world-cups) |
+| ::{ flag=US }:: [ChillierPear](https://osu.ppy.sh/users/9501251) | Organisator und Verwalter des [World Cups](/wiki/Tournaments#offizielle-world-cups) |
```
Comment (respektive) [inline wiki/People/osu!_team/de.md:20]:
```suggestion
| ::{ flag=US }:: [ChillierPear](https://osu.ppy.sh/users/9501251) | Organisator und Verwalter des [World Cups](/wiki/Tournaments#offizielle-world-cups) |
| ::{ flag=US }:: [Death](https://osu.ppy.sh/users/3242450) | [Moderator](/wiki/People/Global_Moderation_Team), technische Unterstützung |
```

File: wiki/People/osu!_team/de.md [hunk 3]
```diff
@@ -18,60 +16,74 @@ Die unten aufgeführten Personen bilden den Kern des **osu!-Teams** und sind hau
...
+| ::{ flag=BR }:: [LeoFLT](https://osu.ppy.sh/users/3668779) | Organisator und Verwalter des [World Cups](/wiki/Tournaments#offizielle-world-cups) |
+| ::{ flag=GB }:: [mangomizer](https://osu.ppy.sh/users/1893718) | [Featured Artist](/wiki/People/Featured_Artists) Kontaktaufnahme, Sekretär beim [World Cup](/wiki/Tournaments#offizielle-world-cups) |
+| ::{ flag=NO }:: [MillhioreF](https://osu.ppy.sh/users/941094) | osu!-Entwickler, technische Unterstützung |
 | ::{ flag=JP }:: [nanaya](https://osu.ppy.sh/users/2387883) |  osu!web Entwickler |
-| ::{ flag=PH }:: [Nathanael](https://osu.ppy.sh/users/2295078) | Allgemeiner Hausmeister, Moderations-Berater |
+| ::{ flag=PH }:: [Nathanael](https://osu.ppy.sh/users/2295078) | Allgemeiner Hausmeister, Moderationsberater |
 | ::{ flag=SE }:: [Naxess](https://osu.ppy.sh/users/8129817) | Berater für die Modding- und Mapping-Szene |
-| ::{ flag=AU }:: [nekodex](https://osu.ppy.sh/users/102) |  osu!web Entwickler, [offizieller osu! Featured Artist](https://osu.ppy.sh/beatmaps/artists/1) |
+| ::{ flag=AU }:: [nekodex](https://osu.ppy.sh/users/102) |  osu!web Entwickler, [osu! Featured Artist](https://osu.ppy.sh/beatmaps/artists/1) |
```
Comment (respektive) [inline wiki/People/osu!_team/de.md:33]:
```suggestion
| ::{ flag=AU }:: [nekodex](https://osu.ppy.sh/users/102) |  osu!web Entwickler, [Featured Artist](https://osu.ppy.sh/beatmaps/artists/1) |
```
English version also dropped the `osu!`

**Review by respektive — CHANGES_REQUESTED** [review]

**Review by respektive — APPROVED** [review]
looks good now

**Pull request merged**